### ISSUE TITLE
Unable to make use of a private service

### ISSUE THREAD

**Issue by williammartin** [storyscript/runtime#535]
### Story

As a developer, if I make use of a private service (that I submitted) in my story
It should be successfully executed on the platform
So that I can make use of my service.

### Justification

Currently, if I submit my own service to the hub, I can't make use of it in a story without an admin making it publicly accessible.

### Acceptance

**Given** a valid microservice in GitHub that has been submitted to the Hub but not been made public

**When** I deploy a story making use of the service and execute it

**Then** it calls the service

**Comment by wilzbach** [comment]
As mentioned on Slack this requires changing the Hub SDK as well, because that's what the compiler currently uses to retrieve the information about hub services.

**Comment by williammartin** [comment]
Yep thanks, I created an issue over on the CLI https://github.com/storyscript/cli/issues/282 which I anticipate results in a hub-sdk issue to fix and then a bump of the dependency over there.
